packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.js

```diff
@@ -90,18 +87,23 @@ export default class QuanticBreadcrumbManager extends LightningElement {
    */
   initialize = (engine) => {
     this.breadcrumbManager = CoveoHeadless.buildBreadcrumbManager(engine);
```

Comment:
We should use `getHeadlessBundle` here as well instead of `CoveoHeadless` directly.

Reply:
done